import.py

```diff
@@ -0,0 +1,7 @@
+import csv
+import os
+import psycopg2
+import psycopg2.extras
+import click
+from flask import current_app, g
+from flask.cli import with_appcontext
```

Comment:
This file doesn't do anything and was probably committed by accident. Best to remove it.

Reply:
@kdmundale fixed in PR #7